fiasco/ions.py

```diff
@@ -283,6 +284,25 @@ def formation_temperature(self) -> u.K:
         """
         return self.temperature[np.argmax(self.ioneq)]
 
+    @property
+    def zeta_0(self) -> u.dimensionless_unscaled:
+        r"""
+        :math: `\zeta_{0}`, the number of vacancies in the ion, which is used to calculate
+        the free-bound Gaunt factor of an ion.
+
+        See Section 2.2 and Table 1 of :cite:t:`mewe_freebound_1986`.
+        """
+        difference = self.atomic_number - (self.charge_state + 1)
+        if difference <= 0:
+            max_vacancies = 1
+        elif difference <= 8 and difference > 0:
+            max_vacancies = 9
+        elif difference <= 22 and difference > 8:
+            max_vacancies = 27
+        else:
+            max_vacancies = 55
+        return float(max_vacancies - difference)
```

Comment:
Does this need to be explicitly cast to a float?

Reply:
Not sure why I did that.  Probably not necessary.